dev-docs/bidders/sharethrough.md

```diff
@@ -27,3 +27,11 @@ The Sharethrough bidder adapter requires additional setup and approval from the
 | `bcat`      | optional | Array of blocked IAB Categories                                                                                                                                                  | `['IAB1-2', 'IAB1-3']`       | `string[]`           |
 | `badv`      | optional | Array of blocked Advertisers by their domains                                                                                                                                    | `['ford.com', 'pepsi.com']`  | `string[]`           |
 | `pkey`      | required | The placement key                                                                                                                                                                | `'DfFKxpkRGPMS7A9f71CquBgZ'` | `string`             |
+| `data`      | optional | Object containing additional first party data.  See [Data Object](#data-object) for details.                                                                        |`data: { pbadlot: 'string' }` | `object`             |
```

Comment:
There's a typo here (pbadlot), but more importantly, it's not ok for you to make pbadslot a bidder-specific param.

You need to be reading this data from the AdUnit.ortb2Imp.ext.data.pbadslot. See https://docs.prebid.org/features/pbAdSlot.html

You can copy that location to wherever your endpoint requires, but you cannot make publishers specify the pbadslot in a special way when the point is standardization.

Reply:
@bretg this is actually how we implemented those features, but I was told in prebid-server#1925 that I had to update documentation.

Is this not the case?